xml/cap_admin_memory_limits.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha-cap-memory-limits"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Container Memory Limits and Requests</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  In &productname;, containers have predefined memory limits and request sizes.
+  Depending on the workload, these may need to be adjusted in some cases.
+ </para>
+ <sect1 xml:id="sec-cap-memory-limits-enable">
+  <title>Enabling and Disabling Memory Limits and Request Sizes</title>
+  <para>
+   By default, memory limits and request sizes are enabled. To disable it, add
+   the following block to your &values-filename; file.
+  </para>
+<screen>features:
+  memory_limits:
+    enabled: false
+</screen>
+  <para>
+   To enable memory limits again, update the above block in your
+   &values-filename; so that <literal>enabled</literal> is set to
+   <literal>true</literal>.
+  </para>
+
+  <!-- Applying change to new vs existing deployments -->
+  &config-new-vs-existing;
+ </sect1>
+ <sect1 xml:id="sec-cap-memory-limits-configure">
+  <title>Configuring Memory Limits and Request Sizes</title>
+  <para>
+   Configuring memory limits and request sizes requires that
+   <literal>feature.memory_limits</literal> is enabled. The default memory limits
+   and request sizes can be found by examining the <literal>resources</literal>
+   block at
+   <link xlink:href="https://github.com/SUSE/kubernetes-charts-suse-com/blob/master/stable/kubecf/config/resources.yaml"/>.
+   To configure memory limits and request sizes, add a
+   <literal>resources</literal> block to your &values-file;. It contains a
```

Comment:
Should this also be &values-filename; as above? It looks a bit odd without the monospaced font.

Reply:
Fixed in https://github.com/SUSE/doc-cap/pull/983/commits/f82b0cf9c8b4e9dcd3cbf9feb27f4b309faa7ac7